SpectralCubeReprojectExample.ipynb

```diff
@@ -0,0 +1,995 @@
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "filename_2 = download_file(\"https://almascience.nrao.edu/dataPortal/member.uid___A001_X87d_X141.a_sma1_sci.spw27.cube.I.pbcor.fits\",\n",
+    "                          cache=True)\n",
+    "filename_2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fuzzy-knife",
+   "metadata": {},
+   "source": [
+    "## Step 2: Load the cubes"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "neutral-devices",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from spectral_cube import SpectralCube"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "seventh-nerve",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "WARNING: StokesWarning: Cube is a Stokes cube, returning spectral cube for I component [spectral_cube.io.core]\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "SpectralCube with shape=(75, 250, 250) and unit=Jy / beam:\n",
+       " n_x:    250  type_x: RA---SIN  unit_x: deg    range:   266.534072 deg:  266.554577 deg\n",
+       " n_y:    250  type_y: DEC--SIN  unit_y: deg    range:   -28.713958 deg:  -28.695975 deg\n",
+       " n_s:     75  type_s: FREQ      unit_s: Hz     range: 139434992275.503 Hz:139503942362.300 Hz"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "cube1 = SpectralCube.read(filename_1)\n",
+    "cube1"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "becoming-translation",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "WARNING: StokesWarning: Cube is a Stokes cube, returning spectral cube for I component [spectral_cube.io.core]\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "SpectralCube with shape=(478, 420, 420) and unit=Jy / beam:\n",
+       " n_x:    420  type_x: RA---SIN  unit_x: deg    range:   266.537002 deg:  266.551600 deg\n",
+       " n_y:    420  type_y: DEC--SIN  unit_y: deg    range:   -28.711371 deg:  -28.698569 deg\n",
+       " n_s:    478  type_s: FREQ      unit_s: Hz     range: 216957714464.027 Hz:217190639088.700 Hz"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "cube2 = SpectralCube.read(filename_2)\n",
+    "cube2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "lined-earthquake",
+   "metadata": {},
+   "source": [
+    "The cubes are at different frequencies - 139 and 89 GHz.\n",
+    "\n",
+    "The first cube covers the H2CS 4(1,3)-3(1,2) line at 139.483699\tGHz.\n",
+    "\n",
+    "The second covers SiO v=5-4 at 217.104984 GHz"
```

Comment:
Add a line on how `find_lines` will return a table for the lines within the freq range of a cube.

Reply:
Either before or after the tables, maybe a note on why multiple lines are returned in the tables, too.